library/pod-security-policy/host-namespaces/samples/psp-host-namespace/constraint.yaml

```diff
@@ -7,3 +7,5 @@ spec:
     kinds:
       - apiGroups: [""]
         kinds: ["Pod"]
+  parameters:
+    exemptImages: ["exception.azurecr.io/exception*"]
```

Comment:
i don't think we need to define these in constraints (especially if it can resolve to a real registry), users can add them if it's needed

Reply:
remove related changes now